components/ide/jetbrains/image/gha-update-image/index.ts

```diff
@@ -126,52 +136,70 @@ const getIDEVersion = function (ide: string) {
     return str.at(-1)!.replace(".tar.gz", "");
 };
 
-const updateLatestVersionsInWorkspaceaAndGradle = async () => {
-    const requests: Promise<any>[] = [];
-    for (const ide of ides) {
-        const params = new URLSearchParams({
-            code: ide.productCode,
-            "release.type": ide.productType,
-            fields: ["distributions", "link", "name", "releases"].join(","),
-            _: Date.now().toString(),
-        });
-
-        const url = new URL(JB_PRODUCTS_DATA_URL);
-        url.search = params.toString();
-        requests.push(axios(url.toString()));
-    }
+// Skip update when build versions are not the same for all IDEs
```

Comment:
@mustard-mh it would be good if we can mock endpoint and write unit tests to capture expectations.

Reply:
@akosyakov not sure if unit tests for this case will be helpful, because it's JetBrains control the response data.

Add integration test (WIP) https://github.com/gitpod-io/gitpod/pull/19665 should be enough?

Reply:
We can record response?